core/assignment.go

```diff
@@ -2,9 +2,24 @@ package core
 
 import (
 	"errors"
+	"fmt"
+	"math"
 	"math/big"
 )
 
+const (
+	percentMultiplier    = 100
+	minChunkLength       = 1
+	maxRequiredNumChunks = 8192
```

Comment:
This would result in the constraint being either tightened or loosened. Either of these can be handled in a non-breaking way. See "Update considerations" [here](https://docs.google.com/document/d/1UwofJjotTgE2OF5ExDfgyODr31KFuVMmCO1OATLmTBU/edit#heading=h.bgl9fiaa9m3r)

Reply:
Can you document in comment how `81921` is decided as a default max?

btw, shall it be named as `maxAllowedNumChunks` or simply `maxNumChunks` which matches `minChunkLength`? (the "max" and "required" together is a bit odd)

Reply:
This doesn't currently constrain the number of chunks; it only limits how many chunks we *Require* the disperser to create. We don't require the disperser to create more than 8192. 

I'll add a comment clarifying and describing the choice of 8192.